### PR TITLE
Create internal link component

### PR DESCRIPTION
**Pull request by adamgoth**
Create a reusable internal link component for consistent implementation

### CODE REVIEW

File: modules/executive/components/ExecutiveOverviewCard.tsx [hunk 1]
```diff
@@ -178,35 +178,33 @@ export default function ExecutiveOverviewCard({
...
-            <Link
-              href={{ pathname: '/executive/[proposal-id]' }}
-              as={{ pathname: `/executive/${proposal.key}` }}
-              passHref
+            <InternalLink
+              href={'/executive/[proposal-id]'}
+              title="View executive details"
+              as={`/executive/${proposal.key}`}
             >
-              <ThemeUILink variant="nostyle" title="View Poll Details" sx={{ width: '100%' }}>
```
Comment (rafinskipg) [inline modules/executive/components/ExecutiveOverviewCard.tsx:196]: does this sx={{ width: '100%' }}> that now is removed affect the UI?

**Review by rafinskipg — APPROVED** [review]

File: modules/executive/components/ExecutiveOverviewCard.tsx [hunk 2]
```diff
@@ -81,17 +81,17 @@ export default function ExecutiveOverviewCard({
...
           <Box>
             <Flex sx={{ flexDirection: 'column', height: '200px' }}>
-              <Link
-                href={{ pathname: '/executive/[proposal-id]' }}
-                as={{ pathname: `/executive/${proposal.key}` }}
-                passHref
+              <InternalLink
+                href={'/executive/[proposal-id]'}
+                title="View executive details"
+                as={`/executive/${proposal.key}`}
```
Comment (b-pmcg) [inline modules/executive/components/ExecutiveOverviewCard.tsx:87]: Nice, great idea. One note, I believe the `as` property is no longer required and we can pass the dynamic route straight into `href`. Check this out for more: https://nextjs.org/docs/api-reference/next/link

Comment (rafinskipg) [reply modules/executive/components/ExecutiveOverviewCard.tsx:87]: yeah

Comment (adamgoth) [reply modules/executive/components/ExecutiveOverviewCard.tsx:87]: > Nice, great idea. One note, I believe the `as` property is no longer required and we can pass the dynamic route straight into `href`. Check this out for more: https://nextjs.org/docs/api-reference/next/link

Sweet, thanks for the heads up. Nice thing is in the future if a change like that occurs again, we'll hopefully only have to update it in one place.

**Pull request merged**